src/main/java/message_search_use_case/MessageSearchInteractor.java

```diff
@@ -24,13 +26,12 @@ public MessageSearchInteractor(MessageSearchGateway gateway, MessageSearchOutput
      */
     @Override
     public MessageSearchResponse search(MessageSearchData data) {
-        //TODO: Implement method (perform checks, etc.)
         if (data.getText().isBlank()) {
             return presenter.prepareFailView("Search query can't be blank.");
         } else if (data.getText().length() <= 5) {
```

Comment:
Why does it have to be more than 5 characters? Is it just an arbitary number to increase search accuracy

Reply:
Yeah, cuz I thought if a person just searches for a comma or something like that, then searches wouldn't be useful. So just to restrict it a bit I thought we could require the user to make their search queries more than 5 characters. Do you think that's too much, or is it ok?

Reply:
Yea, I think that should be fine